### ISSUE TITLE
Delete dialogs - the list of items is incomplete or with wrong syntax

### ISSUE THREAD

**Issue by torchiaf** [rancher/dashboard#13455]
**Setup**
- Rancher version: `head-58aa6e361f`
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

- `Projects/Namespaces` remove: If the items are N and N > 5, there are 2 'and' conjunctions at the end of the message:

  ![Image](https://github.com/user-attachments/assets/4fe6b88d-85ca-44c1-bfe7-aef6b0eca76e)

-  `Pods` remove: Always shows the first 5 elements and the `plusMore` string is missing

  ![Image](https://github.com/user-attachments/assets/ee189475-d89a-4fc9-a44c-09b01ebd8a68)

- Vue3 warnings generated in PromptRemove.vue component

  ![Image](https://github.com/user-attachments/assets/91cd1212-0f96-4fbf-8b71-fa15da62c03d)

**To Reproduce**

**Result**

**Expected Result**

**Screenshots**

**Additional context**

**Comment by a-arias** [comment]
e2e test are sufficient to validate fix.
